llvm/test/Transforms/FunctionSpecialization/literal-const.ll

```diff
@@ -1,7 +1,8 @@
 ; RUN: opt -S --passes="ipsccp<func-spec>" \
+; RUN:        -funcspec-for-literal-constant=false \
 ; RUN:        -force-specialization < %s | FileCheck %s -check-prefix CHECK-NOLIT
 ; RUN: opt -S --passes="ipsccp<func-spec>" \
-; RUN:        -funcspec-for-literal-constant \
+; RUN:        -funcspec-for-literal-constant=true \
```

Comment:
Is it worth removing the `-funcspec-for-literal-constant=true` to defend this being enabled by default?

Reply:
Good point - removed :)